apph-back/README.md

```diff
@@ -0,0 +1,3 @@
+# Apph Backend
+
+Using springboot and maven.
```

Comment:
Peut être update le readme pour préciser comment lancer la back en local

Reply:
Alors le truc c'est que je le lance avec les configurations par default d'intellij (build and run avec java8 sur le dossier apph-back), je ne sais pas si c'est intéressant de le préciser dans le readme.